drivers/gles3/storage/material_storage.cpp

```diff
@@ -2285,6 +2285,57 @@ GLuint MaterialStorage::global_shader_uniforms_get_uniform_buffer() const {
 	return global_shader_uniforms.buffer;
 }
 
+//an alternative way of allocating global shader uniforms
+//allocates a single variable and returns its position in the global shader uniform buffer
+//intended to be used for debug drawing purposes and should not be used for anything substantial
```

Comment:
Our style guide on code comments is to start with a capital and end with a period